templates/global.template

```diff
@@ -0,0 +1,309 @@
+## Just
+
+[![Build Status](https://travis-ci.com/angus-c/just.svg?branch=master)](https://travis-ci.org/angus-c/just.js)
+
+A library of **zero-dependency** npm modules that do just one thing.
+A guilt-free alternative to those bulkier utility libraries. Ideal for PWA development or whenever bytes are precious.
+
+**[Jump To API](#the-modules-package)**
+
+We welcome contributions. Please follow our [contribution guidelines](https://github.com/angus-c/just/blob/master/CONTRIBUTING.md).
+
+## Try :icecream:
+
+A [REPL](https://anguscroll.com/just) for every utility (powered by [RunKit](https://runkit.com))
+
+<a href="https://anguscroll.com/just"><img src="images/repl.png" width="500"/></a>
+
+## Read :books:
+
+- [TRADEOFFS.md](https://github.com/angus-c/just/blob/master/TRADEOFFS.md) -- When to use Just (and when not to).
+- [The Zen of Dependency-Free](https://medium.com/@angustweets/just-a12d54221f65#.ljib0mfr5) -- Why I wrote Just.
+
+## TypeScript <img src="images/ts.png" width="18"/>
+
+We're in the process of adding TypeScript definitions and tests to every Just utility. You're welcome to help us get there!
+[Here's](https://github.com/angus-c/just/pull/247/files) an example PR.
+
+You can verify new TypeScript definitions by running `yarn test-types` (This also gets run as part of the `yarn test` script)
+
+## Browser Support :computer:
+
+Data based on [available saucelabs test browsers](https://github.com/angus-c/just/tree/sauce). It's likely _Just_ is also fully supported by some older versions not verifiable via saucelabs.
+
+| Chrome | Safari | Firefox | Edge | Node | Mobile Safari | Android       |
+| ------ | ------ | ------- | ---- | ---- | ------------- | ------------- |
+| yes    | yes    | yes     | 12   | 6+   | iOS 8+        | Android OS 5+ |
+
+## The Modules :package:
+
+- [Collections](#collections) {}[]
```

Comment:
I guess we could ultimately generate these from `md-variables.json` (if it were grouped by type)

Reply:
Yeah we definitely could. After this PR is merged I also want to resort them alphabetically (within each section) if you agree. Their order in the list of contents and their actual order are currently different for some packages.

Reply:
One issue with alphabetical is it's useful to group like functions together (e.g. `just-camel-case`, `just-snake-case` etc). An alternative is to use the sequence in https://anguscroll.com/just/ as the source of truth.